warehouse/scripts/dbt_artifacts/__init__.py

```diff
@@ -416,21 +429,17 @@ def gvattrs(self) -> Dict[str, Any]:
         """
         Returns a string representation intended for graphviz labels
         """
+        color = "white"
         assert self.node is not None
-        # TODO: do an actual linear transform on this
-        # the top colors are too dark to use as a background
-        white, yellow, orange, red, _, _ = LaJolla_6.hex_colors
+        # TODO: use palettable
         if self.bytes_processed > 500_000_000_000:
```

Comment:
These colors are not mentioned in the legend?

Reply:
these are for run results, not manifest